scripts/build-sample-modules.js

```diff
@@ -35,7 +36,8 @@ const nginxOriginStaticsModulesDir = path.resolve(nginxOriginStaticsRootDir, 'mo
 const pathToNginxOriginModuleMap = path.resolve(nginxOriginStaticsRootDir, 'module-map.json');
 const pathToAssets = path.resolve(sampleProdDir, 'assets');
 const userIntendsToSkipSampleModulesBuild = process.env.ONE_DANGEROUSLY_SKIP_SAMPLE_MODULES_BUILD;
-const archiveBuiltArtifacts = process.argv.includes('--archiveBuiltArtifacts');
+const archiveBuiltArtifacts = argv.archiveBuiltArtifacts || '';
+const bundleStaticsOrigin = argv.bundleStaticsOrigin || 'https://sample-cdn.frank';
```

Comment:
should `'https://sample-cdn.frank'` be the default set up in [`src/server/config/env/argv.js`](https://github.com/americanexpress/one-app/blob/master/src/server/config/env/argv.js)? (similar for `archiveBuiltArtifacts`)
```suggestion
const bundleStaticsOrigin = argv.bundleStaticsOrigin;
```

Reply:
This script is not a part of One App, it is a standalone script used to build the sample modules to run integration tests against. The set up in `src/server/config/env/argv.js` only applies to One App's node process.